consensus/impl/sbp/sbp.go

```diff
@@ -34,7 +35,8 @@ type txExec struct {
 func newTxExec(cdb consensus.ChainDB, bi *types.BlockHeaderInfo) chain.TxOp {
 	// Block hash not determined yet
 	return &txExec{
-		execTx: bc.NewTxExecutor(nil, contract.ChainAccessor(cdb), bi, contract.BlockFactory),
+		// FIXME
+		execTx: bc.NewTxExecutor(context.Background(), nil, contract.ChainAccessor(cdb), bi, contract.BlockFactory),
```

Comment:
Add some description of what to do to this FIXME

Reply:
So the fact that the integration tests are passing may be because there is no timeout check in `SBP` mode?
This only reinforces the need to run those tests on `dpos`
And/or maybe enable the above for `SBP`